tsconfig.json

```diff
@@ -15,6 +15,14 @@
     "lib": [
       "es2017",
       "dom"
-    ]
+    ],
+    "paths": {
+      "parser": [
+        "dist/sketch-parser"
+      ],
+      "parser/*": [
+        "dist/sketch-parser/*"
```

Comment:
this should reference the src folder of the parser.

Reply:
This is probably done by the angular cli. This is default for the libary support if I'm correct. Libary's should be build before you can use it.